data/sql/updates/pending_db_characters/rev_1626661382000836600.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_characters` (`sql_rev`) VALUES ('1626661382000836600');
+
+
+UPDATE acore_world.creature_template SET minlevel = 63,maxlevel=63,HealthModifier=180.882  where entry IN (4949, 10181, 10540, 3057, 16802, 29611, 2784, 7937, 7999, 17468)
```

Comment:
Hello an thx for ylur contribution! Please read https://www.azerothcore.org/wiki/sql-standards

- No need to call DB Name here
- Add backticks ` around column names 
- Write keywords in uppercase
- Semicolon is missing at the end

Reply:
Hey, sorry what do you mean by "No need to call DB Name here"? Putting the DB name into a variable?